src/main/java/uk/gov/hmcts/reform/authorisation/validators/ServiceAuthTokenValidator.java

```diff
@@ -1,17 +1,23 @@
 package uk.gov.hmcts.reform.authorisation.validators;
 
 import feign.FeignException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
 import uk.gov.hmcts.reform.authorisation.exceptions.InvalidTokenException;
 import uk.gov.hmcts.reform.authorisation.exceptions.ServiceException;
 
 import java.util.Collections;
 import java.util.List;
 
+@Component
+@ConditionalOnProperty("idam.s2s-authorised.services")
 public class ServiceAuthTokenValidator implements AuthTokenValidator {
 
     private final ServiceAuthorisationApi api;
 
+    @Autowired (required = true)
```

Comment:
isn't required the default?
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/annotation/Autowired.html

```suggestion
    @Autowired
```